src/CoreBundle/Doctrine/Type/ObjectType.php

```
@@ -0,0 +1,68 @@
+    {
+        return $platform->getClobTypeDeclarationSQL($column);
+    }
+
+    #[Override]
+    public function convertToDatabaseValue(mixed $value, AbstractPlatform $platform): string
+    {
+        return serialize($value);
+    }
+
+    #[Override]
+    public function convertToPHPValue(mixed $value, AbstractPlatform $platform): mixed
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$platform'. (undefined)

(UnusedFormalParameter)